### ISSUE TITLE
October - December Sprint Meta Issue

### ISSUE THREAD

**Issue by crangelsmith** [alan-turing-institute/heartfm-data#2]
Goal of this sprint: 

**Pre-train and benchmark foundation models using open datasets on Isambard.**

0. Decide which open dataset we will use for pre-training #1 
1. Use [CINEMA](https://github.com/mathpluscode/CineMA) pre-training scripts to pre-train on the dataset decided in 0.
2. Write our own Vanilla Vision Transformer (call it HeartFM?) and pre-train it on the dataset decided at 0. Use reconstruction loss error to evaluate the performance of models 1 and 2.
3. Evaluate and benchmark 1 and 2 on fine-tuning tasks from Cinema. Compare with Cinema open weights (trained on Biobank) to estimate the loss of performance as a function of the reduced dataset size. 
4. If we have time, we could add a second dataset to 1. and 2. with continuous pre-training and benchmark improvement in performance and the effect of continuous pre-training.
